src/d3a/models/strategy/area_agents/one_sided_engine.py

```diff
@@ -236,10 +233,11 @@ class BalancingEngine(IAAEngine):
 
     def _forward_offer(self, offer, offer_id):
         forwarded_balancing_offer = self.markets.target.balancing_offer(
-            offer.price + (offer.price * (self.transfer_fee_pct / 100)),
+            offer.price,
             offer.energy,
             self.owner.name,
-            True
+            True,
```

Comment:
I know it wasn't there before, but let's be more explicit here and change it to `from_agent=True` for the readability of the code.  Thanks!